generators/server/index.js

```diff
@@ -194,6 +194,10 @@ module.exports = JhipsterServerGenerator.extend({
                     this.languages = ['en', 'fr'];
                 }
 
+                if(this.applicationType == 'gateway') {
```

Comment:
can we do this in `app/index.js` itself similar to how we set it for microservice